src/hooks/useTikTokAuth.ts

```diff
@@ -62,11 +62,13 @@ export const useTikTokAuth = ({
       message.send()
       const response = await message.receive()
 
-      const { accessToken, openId, expiresIn } = response
+      const { accessToken, openId, expiresIn, error } = response
       if (accessToken && openId && expiresIn) {
         storeAccessToken(accessToken, openId, expiresIn, callback)
       } else {
-        onError(new Error('Native mobile TikTok auth failed'))
+        onError(
+          error || new Error('Access token not returned from native layer')
```

Comment:
should this be new Error(error || 'Acces....') that would match below

Reply:
Yup, good catch